docs/developer/extending/apps/asynchronous-webhooks.mdx

```diff
@@ -4,7 +4,9 @@ title: Asynchronous Webhooks
 
 ## Architecture
 
-Asynchronous webhooks can be used for reacting to the events emitted by Saleor. The worker instance will send the event payload via the chosen protocol. Your App should acknowledge receiving that payload. The result of the App operations is not sent back to the Saleor. If you need to send data back to the user (for example, calculated shipping cost), you should use a [synchronous webhook](developer/extending/apps/synchronous-webhooks/key-concepts.mdx).
+Asynchronous webhooks can be used for reacting to the events emitted by Saleor.
+
+The worker instance will send the event payload via the chosen protocol. Your App should acknowledge receiving that payload. The result of the App operations is not sent back to the Saleor. If you need to send data back to the user (for example, calculated shipping cost), you should use a [synchronous webhook](developer/extending/apps/synchronous-webhooks/key-concepts.mdx).
```

Comment:
Maybe "Your App should acknowledge receiving that payload, by responding with HTTP 200 status"